src/views/viewer/ChannelView/ChannelView.style.tsx

```diff
@@ -126,15 +133,82 @@ export const PaginationContainer = styled.div`
 `
 
 export const TabsContainer = styled.div`
+  display: grid;
   margin-bottom: ${sizes(8)};
-  border-bottom: solid 1px ${colors.gray[800]};
-
+  gap: ${sizes(2)};
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: 1fr 1fr;
+  grid-template-areas:
+    'tabs tabs tabs'
+    'search sort sort';
+  align-items: baseline;
   ${media.compact} {
     padding-top: ${sizes(8)};
   }
+  ${media.small} {
+    border-bottom: solid 1px ${colors.gray[800]};
+    grid-template-areas:
+      'tabs tabs '
+      'search  sort';
+  }
+  ${media.medium} {
+    grid-template-areas: initial;
+    gap: ${sizes(8)};
+    grid-template-rows: 1fr;
+    grid-template-columns: auto 1fr 250px;
+  }
+`
+
+export const SearchContainer = styled.div`
+  display: flex;
+  grid-area: search;
+  width: 100%;
+  align-items: center;
+
+  ${media.base} {
+    align-self: end;
+  }
+  ${media.small} {
+    align-self: initial;
+  }
+  ${media.medium} {
+    grid-area: initial;
+  }
+`
 
+export const StyledTabs = styled(Tabs)`
+  grid-area: tabs;
+  ${media.base} {
+    border-bottom: solid 1px ${colors.gray[800]};
+  }
+  ${media.small} {
+    border-bottom: none;
+  }
   ${media.medium} {
-    display: grid;
-    grid-template-columns: 1fr 250px;
+    grid-area: initial;
   }
 `
+
+type TextFieldProps = {
+  isOpen?: boolean
+}
+export const StyledTextField = styled(TextField)<TextFieldProps>`
+  transition: all ${transitions.timings.regular} ${transitions.easing};
+  will-change: max-width;
+  width: 100%;
+  align-items: center;
+  max-width: ${({ isOpen }) => (isOpen ? '192px' : '0px')};
```

Comment:
Input is much bigger in the design 
![image](https://user-images.githubusercontent.com/51168865/126761604-3e32b06c-5dad-4f83-a446-e5fb9318acf8.png)

Reply:
I think thats an old design:
https://www.figma.com/file/TGUpBXgXf9ceMvHbRB5BIx/Joystream-Atlas?node-id=10474%3A305579